lib/chibios-syscalls/cpp_stubs.cpp

```diff
@@ -10,55 +11,33 @@ int __cxa_guard_acquire(__guard*);
 void __cxa_guard_release(__guard*);
 void __cxa_guard_abort(__guard*);
 void __cxa_pure_virtual();
-void* __dso_handle = NULL;
+void* __attribute__((weak)) __dso_handle = NULL;
 void _exit(int status);
 pid_t _getpid(void);
 int _kill(int pid, int sig);
 }
 
-void __cxa_pure_virtual()
+void __attribute__((weak)) __cxa_pure_virtual()
```

Comment:
Why make them as weak linking ? I think the libc++ version is already weak, so what is the point ?

Reply:
was still looking at that. Our build is messier than I thought, so it took a while to fix it for all boards. Let's see if Travis is happy :crossed_fingers: